pkg/cli/prompt/validator.go

```diff
@@ -22,9 +22,10 @@ import (
 )
 
 const (
-	invalidNamespaceNameMessage = "namespace must be 1-63 characters, made up of of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character"
-	invalidResourceNameMessage  = "name must be made up of alphanumeric characters and hyphens, and must begin with an alphabetic character and end with an alphanumeric character"
-	invalidUUIDv4Message        = "must be a valid UUID v4 (GUID)"
+	invalidNamespaceNameMessage   = "namespace must be 1-63 characters, made up of of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character"
```

Comment:
```suggestion
	invalidNamespaceNameMessage   = "namespace must be 1-63 characters, made up of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character"
```

Reply:
This is kubernetes error message: and must start and end with an alphanumeric character.